Formula/libimobiledevice.rb

```diff
@@ -20,20 +20,76 @@ class Libimobiledevice < Formula
     depends_on "libxml2"
   end
 
+  option "with-python", "Enable Cython Python bindings"
+  option "with-python3", "Enable Cython Python3 bindings"
+
   depends_on "pkg-config" => :build
   depends_on "libtasn1"
-  depends_on "libplist"
   depends_on "usbmuxd"
   depends_on "openssl"
+  depends_on "libplist" => (build.with?("python") ? "with-python" : []) || (build.with?("python3") ? "with-python3" : [])
+  depends_on :python => :optional
+  depends_on :python3 => :optional
+
+  resource "cython" do
+    url "http://cython.org/release/Cython-0.24.tar.gz"
```

Comment:
This is failing: http://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/4177/
And locally:
```
🌲  curl -IL http://cython.org/release/Cython-0.24.tar.gz
HTTP/1.1 404 Not Found
Server: GitHub.com
Date: Sun, 03 Jul 2016 19:43:00 GMT
Content-Type: text/html; charset=utf-8
Content-Length: 9340
ETag: "5745e9c5-247c"
Access-Control-Allow-Origin: *
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; img-src data:; connect-src 'self'
X-GitHub-Request-Id: 688317F6:270F:4678B70:57796AC4